tests/e2e/scripts/verify-state.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
Is there a way to avoid this script-based verification ?

Reply:
Updated with assert yaml file